examples/flower-client-authentication/README.md

```diff
@@ -0,0 +1,80 @@
+# Flower Client Authentication with PyTorch 🧪
+
+> 🧪 = This example covers experimental features that might change in future versions of Flower
+> Please consult the regular PyTorch code examples ([quickstart](https://github.com/adap/flower/tree/main/examples/quickstart-pytorch), [advanced](https://github.com/adap/flower/tree/main/examples/advanced-pytorch)) to learn how to use Flower with PyTorch.
+
+The following steps describe how to start a long-running Flower server (SuperLink) and then run a Flower App (consisting of a `ClientApp` and a `ServerApp`) with client authentication enabled.
+
+## Preconditions
+
+Let's assume the following project structure:
+
+```bash
+$ tree .
+.
+├── client.py         # <-- contains `ClientApp`
+├── server.py         # <-- contains `ServerApp`
+├── task.py           # <-- task-specific code (model, data)
+└── requirements.txt  # <-- dependencies
+```
+
+## Install dependencies
+
+```bash
+pip install -r requirements.txt
+```
```

Comment:
Can we move to the new example dependency management approach introduced by @chongshenng?

Reply:
@danieljanes @danielnugraha I've modified this example to use the `hatchling` approach here #3377. Feel free to merge into this PR if you're happy with it!